packages/cookies-tooltip/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const COOKIE_ALLOWED_KEY = 'cookie_allowed'
```

Comment:
enum or object can be used here instead (to make types harder)

Reply:
not sure to use it here